python/requirements/requirements_extra.txt

```diff
@@ -0,0 +1,3 @@
+# TODO: Remove this file and move to requirements.txt once https://github.com/ray-project/ray/pull/29375 is merged.
```

Comment:
Since https://github.com/ray-project/ray/pull/29375 is also a release blocker - should we wait until that is merged and resolved this? 
cc @krfricke